app/snapshot.py

```diff
@@ -153,27 +151,3 @@ def included_in_monthly_price(self):
             return True
         else:
             return False
-
-
-# Estimated monthly costs were formulated by taking the average monthly costs of N. California and Oregon
-def estimate_monthly_snapshot_price(type: str, size: float) -> float:
```

Comment:
I'd consider leaving these in snapshot.py - I think that is a better fit for them